docs/troubleshooting/installation.md

```diff
@@ -25,13 +25,48 @@ default via 10.10.0.10 dev harvester-mgmt proto dhcp        <-- Does a default r
 
 - Check that your DHCP server offers a default route option. Attaching content from `/run/cos/target/rke2.log` is helpful too.
 
+## Modifying cluster token on agent nodes
+
+For agent nodes to join a cluster, they must have a cluster token identical to the one set up by the
+server node. To see whether nodes are unable to join a cluster is caused by unmatched token,
+SSH into the agent nodes and check the log of `rancherd` service:
```

Comment:
```suggestion
When an agent node fails to join the cluster, it can be related to the cluster token not being identical to the server node token.
In order to confirm the issue, connect to your agent node (i.e. with SSH) and check the `rancherd` service log with the following command:
```